R/read.quitte.R

```diff
@@ -59,7 +59,10 @@ read.quitte <- function(file,
                              drop.na, comment) {
 
         if (grepl("\\.xlsx?$", f)) {
-            return (as.quitte(read_excel(path = f, sheet = if ('data' %in% excel_sheets(f)) 'data' else 1)))
+            data <- read_excel(path = f, sheet = if ('data' %in% excel_sheets(f)) 'data' else 1)
+            periods <- names(data)[grep("^[0-9]{4}$", names(data))]
+            data <- as.quitte(pivot_longer(data, all_of(periods), names_to = 'period', values_drop_na = drop.na))
```

Comment:
That's quite long-winded …  why not `pivot_longer(matches("^[0-9]{4}$")`?

Reply:
Well, I copied that from [this line](https://github.com/pik-piam/quitte/blob/21132a41eac8e291e201364182d4b6611f5e2b9e/R/read.quitte.R#L105-L108)…